CyuVR/CyuLoaderVR.cs

```diff
@@ -1,10 +1,11 @@
 ﻿using BepInEx;
-using Harmony;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using UnityEngine;
 using System.ComponentModel;
+using Harmony;
+using BepInEx4;
```

Comment:
What and why is this additional dependency added here?
It does not compile on my end.

Reply:
Have you updated your BepInEx assembly files? It seems like this project was originally made with BepInEx v4 in mind, but if you use the latest version, the `ConfigWrapper<>` type is undefined. By adding the BepInEx4 dependency from the [BepInEx4Upgrader](https://github.com/BepInEx/BepInEx.BepInEx4Upgrader) assembly, we can build it with modern versions of BepInEx.

Reply:
Right, since this plugin targets BepInEx 4, I just build it with BepInEx 4 assemblies and let BepInEx4Upgrader handle the patching at runtime, as how it's designed. 
Although, that does require having to reference an older version of ConfiurationManager that is compatible with BepInEx 4.